src/inc/til/coalesce.h

```diff
@@ -22,6 +34,15 @@ namespace til
         return T{};
     }
 
+    // Method Description:
+    // - Base case provided to throw an assertion if you call coalesce_value(opt, opt, opt)
+    template<typename T>
+    T coalesce_value(const winrt::Windows::Foundation::IReference<T>& base)

```

Comment:
til cannot have an unguarded reference to `IReference`. `til` is conceptually below things that use winrt.

Reply:
oh i see, you.. did the thing i did in JsonUtils. Clever.